<launch>
  
	<!-- Abnormals configuration -->
	<arg name="abnormals_period" default="10" doc="period" />
		
	<!-- UDP communication to server -->
	<arg name="local_id" default="2" doc="local id (KIRO: 1, REDONE: 2)" />
	<arg name="data_type" default="2" doc="data type (MAP: 1, ABNORMAL: 2, IMAGE: 3)" />
	<arg name="robot_ids" default="[1, 2, 3, 4, 5, 6, 7, 8, 9]" doc="agent id list" />
	
	<arg name="agent_abnormals_topic" default="/osr/agent_abnormals" doc="agent_abnormals_topic" />
	
  	<node pkg="osr_demos" type="abnormals_receiver_multiport" name="abnormals_receiver_multiport" output="screen">
		<param name="period" value="$(arg abnormals_period)" />
		<param name="agent_abnormals_topic" value="$(arg agent_abnormals_topic)" />
        <param name="local_id" value="$(arg local_id)" />
        <param name="data_type" value="$(arg data_type)" />
		<rosparam param="robot_ids" subst_value="True">$(arg agent_ids)</rosparam>
  	</node>
</launch>